code/controllers/subsystem/explosions.dm

```diff
@@ -268,7 +268,7 @@ SUBSYSTEM_DEF(explosions)
 					var/echo_volume = 40
 					if(devastation_range)
 						baseshakeamount = devastation_range
-						shake_camera(M, 10, clamp(baseshakeamount*0.25, 0, 2.5))
+					shake_camera(M, 10, clamp(baseshakeamount*0.25, 0, 2.5))
```

Comment:
I'm not sure this works considering the `echo_volume = 60` then goes up an indent.

Can you test this next try?

Reply:
Yes, I mixed up the change there. The camera shake should only happen when there is devastation range, e.g. a big explosion enough to break through the hull. Otherwise there is no shake.

Reply:
After testing this I can confirm it works as intended.